descriptors/ModuleDescriptor-template.json

```diff
@@ -851,6 +851,17 @@
       "visible": false
     }
   ],
+  "metadata": {
+    "user": {
+      "type": "system",
+      "permissions": [
+        "inventory-storage.all",
+        "source-records-manager.all",
+        "source-storage.all",
+        "converter-storage.all"
```

Comment:
propose to use limited list of permissions:
"source-storage.snapshots.item.get",
"source-storage.snapshots.collection.get",
"source-storage.snapshots.put",
"users.collection.get",
"inventory-storage.identifier-types.collection.get",
"inventory-storage.classification-types.collection.get",
"inventory-storage.instance-types.collection.get",
"inventory-storage.electronic-access-relationships.collection.get",
"inventory-storage.instance-formats.collection.get",
"inventory-storage.contributor-types.collection.get",
"inventory-storage.contributor-name-types.collection.get",
"inventory-storage.instance-note-types.collection.get",
"inventory-storage.alternative-title-types.collection.get",
"inventory-storage.modes-of-issuance.collection.get",
"inventory-storage.item-damaged-statuses.collection.get",
"inventory-storage.instance-statuses.collection.get",
"inventory-storage.nature-of-content-terms.collection.get",
"inventory-storage.instance-relationship-types.collection.get",
"inventory-storage.holdings-types.collection.get",
"inventory-storage.holdings-note-types.collection.get",
"inventory-storage.holdings-sources.collection.get",
"inventory-storage.ill-policies.collection.get",
"inventory-storage.call-number-types.collection.get",
"inventory-storage.statistical-codes.collection.get",
"inventory-storage.statistical-code-types.collection.get",
"inventory-storage.item-note-types.collection.get",
"inventory-storage.authority-note-types.collection.get",
"inventory-storage.authority-source-files.collection.get",
"inventory-storage.material-types.item.get",
"inventory-storage.material-types.collection.get",
"inventory-storage.loan-types.item.get",
"inventory-storage.loan-types.collection.get",
"inventory-storage.locations.item.get",
"inventory-storage.locations.collection.get",
"inventory-storage.subject-types.collection.get",
"inventory-storage.subject-sources.collection.get",
"inventory-storage.instance-date-types.collection.get",
"converter-storage.field-protection-settings.item.get",
"converter-storage.field-protection-settings.collection.get",
"configuration.entries.collection.get",
"isbn-utils.convert-to-13.get",
"instance-authority-links.instances.collection.get",
"instance-authority-links.instances.collection.put",
"instance-authority.linking-rules.collection.get",
"source-storage.snapshots.post"